source/blog/2018-12-03-ember-cli-guides.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Launch of the new Ember CLI Guides
+author: Jen Weber
+tags: Announcement
+responsive: true
+---
+
+The Ember Learning Team is excited to announce the launch of the new and improved [Ember CLI Guides](https://cli.emberjs.com)! So excited, in fact, that we want to share a case study of the tech behind the new site and open source project management. This release is the main part of the work for ember-cli [RFC #120](https://github.com/ember-cli/rfcs/pull/120).
+
+## What has changed?
+
+What has changed with the launch of the new CLI Guides?
+
+1. There is brand-new content, like a tutorial to help you write your first addon
+2. The CLI Guides are now an Ember app, using the [Guidemaker](https://github.com/empress/guidemaker) addon
+3. The branding is consistent with the rest of the [emberjs.com](https://emberjs.com) site, thanks to the use of [ember-styleguide]() components and stylesheets
+4. A _significant_ portion of the past content has been revised or rewritten. Outdated information is gone, and modern best practices are represented better
+
+## Why was this project needed?
+
+Like many Ember documentation apps of the past, the Ember CLI Guides were not an Ember app. Over the past two years, the Learning Team has been Emberifying such sites, making them easier to maintain and develop further. Now, the only one remaining is the [website](https://github.com/emberjs/website) repository, which contains static pages like the home page and the blog. The transformation is [currently underway](https://github.com/ember-learn/ember-website).
+
+This CLI Guides project tackled a few additional critical issues, like correcting outdated content and revising the learning flow. The original CLI guides were written a long time ago, before the CLI itself was fully formed, so it is natural that some rearranging was needed.
+
+## Creating apps with Guidemaker and Ember-Styleguide
+
+With as many apps as the Learning Team manages, it's helpful to have a consistent structure, look, and feel. For the most part, this project could focus on written content rather than regular web app concerns. It was awesome! We did this by using Guidemaker and ember-styleguide.
+
+[Guidemaker](https://github.com/empress/guidemaker) is an Ember Addon originally created by [Chris Manson](https://github.com/mansona). It helps quickly create sites with static content. It also handles versioned documentation, like that found in the [Ember Guides](https://guides.emberjs.com). Guidemaker uses Broccoli's asset compilation pipeline to turn a tree of Markdown files into HTML that can be served statically. By using Guidemaker, if we need to version the CLI docs in the future, that feature is ready to go overnight. It also means that contributors to one Guide should feel right at home helping with others.
```

Comment:
```suggestion
[Guidemaker](https://github.com/empress/guidemaker) is an Ember Addon originally created by [Chris Manson](https://github.com/mansona). It helps quickly create sites with static content. It also handles versioned documentation, like that found in the [Ember.js Guides](https://guides.emberjs.com). Guidemaker uses Broccoli's asset compilation pipeline to turn a tree of Markdown files into HTML that can be served statically. By using Guidemaker, if we need to version the CLI docs in the future, that feature is ready to go overnight. It also means that contributors to one Guide should feel right at home helping with others.
```

Should we just change `Ember Guides` to `Ember.js Guides` for consistency since the rest of the blog uses the latter naming